managed/build.sbt

```diff
@@ -168,7 +168,8 @@ libraryDependencies ++= Seq(
   "com.fasterxml.jackson.core" % "jackson-core" % "2.10.5",
   "com.jayway.jsonpath" % "json-path" % "2.4.0",
   "commons-io" % "commons-io" % "2.8.0",
-  "commons-codec" % "commons-codec" % "1.15"
+  "commons-codec" % "commons-codec" % "1.15",
+  "org.apache.commons" % "commons-collections4" % "4.4"
```

Comment:
We already use guava that has all the necessary collection required by this change. 
Can you please use guava and not add yet another dependency?
@bhavin192

Reply:
Sure, I wasn't aware of that. I will work on removing that and using guava. I will create a new PR shortly.